### ISSUE TITLE
Merge solving code into main file so that project can run without any tweaking

### ISSUE THREAD

**Issue by ArDee20p** [ArDee20p/MATH482-ProgrammingProjectA#3]
Right now, insane.c causes the project to fail to compile if left included in the project in VS.
This code should be merged into the main .cpp file so that everything is neat.
Once @benburrill has refined their algorithm as much as they can, it would be nice to be able to merge it. Not sure how doable that is given that the main is C++ and Ben's solution is in C, however.

**Comment by benburrill** [comment]
Is it failing because there's multiple main functions?  Surely there's a way to make VS build multiple executables.

**Comment by ArDee20p** [comment]
could try adding it as a separate project under the same solution, i suppose

**Comment by benburrill** [comment]
Yeah, that sounds like it might work.  Personally I'm not using VS, I'm just compiling it directly with gcc.
